### DISCUSSION TITLE
How should I wire the TFT panel with D1 Mini withouth a shield? (piotrwawrzyn / bitcoin-watch)

### DISCUSSION THREAD

**Discussion by sztamas1990** [piotrwawrzyn/bitcoin-watch#6] [Q&A]
Hi,

Can you help me? I dont have a shield yet, but i would like to test your code.
Can you share a wiring diagram with me, how schould i wire the 240x240 TFT panel directly to the Wemos D1 mini?

Thanks for your reply!

Regards,
Tamas

**Comment by piotrwawrzyn** [comment]
Hmm, I don't have such diagram.

@witnessmenow could you please address this?

**Reply by witnessmenow** [reply]
Sure!

@sztamas1990 the wiring that I use for the shield is here:

https://github.com/witnessmenow/D1-Mini-TFT-Shield#display

You should be able to just wire a ST7889 module up using the same wiring and the code should work the same

**Comment by sztamas1990** [comment]
Thanks a lot finally it works!

And I'm facing with another problem: display works well, but coingecko api seems not working.
I receive the following errors on port monitor:

Error connecting to API while downloading base data
-1
Error connecting to API while downloading BTC and ETH data
-1


Can you help me to solve these problems, please?

Thanks a lot!

Regards,
Tamas

**Reply by winner123421** [reply]
Hi,
This may or may not work for you but for me I changed the coingeckoSslFingerprint to "33 C5 7B 69 E6 3B 76 5C 39 3D F1 19 3B 17 68 B8 1B 0A 1F D9" and it started working.

![Capture](https://user-images.githubusercontent.com/75743503/146758697-32924d67-cc4b-4cc2-b0d4-8ccb55094d4f.PNG)


Thank you.
Best of Regards,
winner

**Comment by sztamas1990** [comment]
Thanks a lot!

Now it works perfectly!
Can u tell me how can I refresh the fingerprint by myself next time?

Thanks for your help!
Regards,
Tamas

winner123421 ***@***.***> ezt írta (időpont: 2021. dec. 20.,
H, 12:14):

**Reply by winner123421** [reply]
No problem!

Here is a link that you could use to generate fingerprint
https://www.grc.com/fingerprints.htm

Thank you.
Best Regards,
winner
